gms_app/models.py

```diff
@@ -2,36 +2,84 @@
 
 
 class Garden(models.Model):
-    name = models.CharField(max_length=50)
-    description = models.TextField(blank=True)
+    name = models.CharField(max_length=50, verbose_name="Garden Name", help_text="Enter the name of the garden")
```

Comment:
Took the liberty to add naming and help text to the model fields and also the meta in the models